lib/screens/repositories_screen.dart

```diff
@@ -22,12 +22,13 @@ class _RepositoriesScreenState extends State<RepositoriesScreen> {
     "TypeScript",
   ];
   String? _selectedLanguage;
-  List<String> repositories = [];
+  List<String> _repositories = [];
 
-  void _didSelectItem(String item) {
+  void _didSelect(String item) {
     setState(() {
       _selectedLanguage = item;
-      repositories = List<String>.generate(100, (index) => "Repo ${index + 1}");
+      _repositories =
+          List<String>.generate(100, (index) => "Repo ${index + 1}");
```

Comment:
When the line length is exceeded, please try make it looke cleaner by adding a comma at the end and re-align via IDE.

Reply:
done